EcoSoapBank/Pickups/PickupsView.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  PickupsView.swift
+//  EcoSoapBank
+//
+//  Created by Jon Bash on 2020-08-10.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
+
+import SwiftUI
+
+
+protocol PickupsViewDelegate: AnyObject {
+    func logOut()
+}
+
+
+struct PickupsView: View {
+    @ObservedObject private var pickupController: PickupController
+
+    @State private var makingNewPickup = false
+
+    private weak var delegate: PickupsViewDelegate?
+
+    init(
+        pickupController: PickupController,
+        delegate: PickupsViewDelegate?
+    ) {
+        self.pickupController = pickupController
+        self.delegate = delegate
+    }
+
+    var body: some View {
+        NavigationView {
+            PickupHistoryView(pickupController: pickupController)
+                .navigationBarTitle("Pickup History", displayMode: .inline)
+                .navigationBarItems(
+                    leading: Button(
+                        action: { self.delegate?.logOut() },
+                        label: { Text("Log out") }),
```

Comment:
Will we have this logout button on just the pickups screen for now? Will it move to the profile/settings tab later? Or should we think about implementing the settings tab off the bat even though we aren't doing anything with the profiles?

Reply:
I'll remove this logout button in a future commit/PR.